src/Compilers/CSharp/Portable/CSharpCodeAnalysis.csproj

```diff
@@ -832,13 +59,7 @@
     <BoundTreeDefinition Include="BoundTree\BoundNodes.xml">
       <SubType>Designer</SubType>
     </BoundTreeDefinition>
-    <Compile Include="Generated\BoundNodes.xml.Generated.cs" />
```

Comment:
@sharwell is correct. We also have a Jenkins test to fail CI if the generated files and source XML get out of sync.